src/classes/ADDR_Contact_TDTM.cls

```diff
@@ -265,38 +266,44 @@ public class ADDR_Contact_TDTM extends TDTM_Runnable {
     * @param listCon the list of Contacts to update
     * @return void  
     ********************************************************************************************************/
-    private void setConAddrFromLookup(list<Contact> listCon, DmlWrapper dmlWrapper, TDTM_Runnable.Action triggerAction) {
-    	
-    	list<Contact> listConCreateAddr = new list<Contact>();
-    	set<Id> setAddrId = new set<Id>();
-    	for (Contact con : listCon) {
-    		if (con.Current_Address__c != null)
-    		  setAddrId.add(con.Current_Address__c);
-    	}
-    
-        map<Id, Address__c> mapAddrIdAddr = new map<Id, Address__c>([select Id, Address_Type__c, Household_Account__c, 
-            MailingStreet__c, MailingStreet2__c, MailingCity__c, MailingState__c, Geolocation__Latitude__s, Geolocation__Longitude__s, 
-            MailingPostalCode__c, MailingCountry__c from Address__c where Id in :setAddrId]);
+    private void setConAddrFromLookup(List<Contact> listCon, DmlWrapper dmlWrapper, TDTM_Runnable.Action triggerAction) {
+
+        List<Contact> listConCreateAddr = new List<Contact>();
+        Set<Id> setAddrId = new Set<Id>();
+        for (Contact con : listCon) {
```

Comment:
Good suggestion. I ran some quick performance tests against a loop of 10000 iterations. The difference between the two was so tiny that it could just be Salesforce's platform.